### DISCUSSION TITLE
Multiple download locations

### DISCUSSION THREAD

**Discussion by itbeemer** [jcwillox/hass-auto-backup#42] [Q&A] [answered]
Good afternoon,
Is it possible to have this add-in use multiple download locations?
I am using HA on an RPi.  I would like to have to go to the normal backups folder, a usb stick locally connected to the RPi, and to a samba share.  (for redundancy)

Thanks much!
Barak

**Comment by jcwillox** [comment, marked as answer]
Do you mean like make `download_path` able to accept a list of locations and download the backup to each of them, that would be very easy to implement.

**Reply by itbeemer** [reply]
Correct, use download_path and be able to point to multiple targets such as default location | /dev/sda1/hassio,bak | smb://router/backups/hassio (the smb location is open, in my implementation, but I dont know how you would deal when un/pw necessary for other implementations)

**Reply by jcwillox** [reply]
Sounds good I'll convert this into a feature request.
Yeah, the whole `download_path` thing is not meant to be very complex, it's really just there for convenience for the people who can find a good way to use it, for more complex operations there are better tools that can actually access the `/backup` folder like [jcwillox/hassio-rclone-backup](https://github.com/jcwillox/hassio-rclone-backup).
